appveyor.yml

```diff
@@ -1,6 +1,6 @@
 environment:
   matrix:
-    - TOXENV: style
+    - TOXENV: fix-lint
```

Comment:
hmmm, I don't like this name, cause it does not only fixes but checks style :thinking: